retrievalmarket/server/gsunpaidretrieval.go

```diff
@@ -175,7 +176,7 @@ func (g *GraphsyncUnpaidRetrieval) CancelTransfer(ctx context.Context, id datatr
 
 	if state == nil {
 		g.activeRetrievalsLk.Unlock()
-		return fmt.Errorf("no transfer with id %d", id)
+		return fmt.Errorf("%w with id: %d", ErrRetrievalNotFound, id)
```

Comment:
I think the error needs to be at the end in order for it to be wrapped properly:
return fmt.Errorf("transfer id %d: %w", id, ErrRetrievalNotFound)